tyk-control-plane/values.yaml

```diff
@@ -509,6 +509,11 @@ tyk-gateway:
     #     readOnly: true
     extraVolumeMounts: []
 
+    podAnnotations: {}
+
+    # podLabels specifies labels to be added in gateway Pod
+    podLabels: {}
```

Comment:
Please add comments about how to use these fields and what they do. We have a few comments for them [here.](https://github.com/TykTechnologies/tyk-charts/blob/c6c8f1b58ffd0c8e50c59eddb66a9a0cfaef02c2/components/tyk-gateway/values.yaml#L195-L205) 
also, what do you think about moving these fields to upper as they appear in gateway component chart?